pkgs/tools/filesystems/ceph/default.nix

```diff
@@ -2,7 +2,7 @@
 , ensureNewerSourcesHook
 , cmake, pkg-config
 , which, git
-, boost, python3Packages
+, boost, python
```

Comment:
Isn't this changing the python version?

Reply:
No, this PR causes zero rebuilds. But this lead to an unused argument which I now removed.